lib/devices/cpu_device_manager.js

```diff
@@ -51,12 +51,15 @@ export default class CpuDeviceManager {
 
 class CpuDevice {
   constructor(id, active=false) {
-    this.id = id
+    this.id = id;
```

Comment:
Not at all important.. .but I would either do the string conversion here.
Or even better `assert(typeof(id) === 'string')`